app/src/components/common/TmPage.vue

```diff
@@ -22,14 +22,16 @@
       :subtitle="emptySubtitle"
     />
 
-    <slot v-else-if="!loading && !empty"></slot>
+    <template v-else-if="!loading && !empty">
+      <slot></slot>
+      <TmDataLoading v-if="!loading && loadingMore" />
```

Comment:
```suggestion
      <TmDataLoading v-if="loadingMore" />
```

Reply:
then you get two different loaders at the beginning when opening the notifications page